.ci/load/Jenkinsfile

```diff
@@ -0,0 +1,170 @@
+// For documentation on this pipeline, please see the README.md in this directory
+pipeline {
+    agent any
+    environment {
+        REPO = 'apm-agent-java'
+        APP = 'spring-petclinic'
+        APP_BASE_DIR = "src/${APP}"
+        METRICS_BASE_DIR="metrics/"
+        AGENT_BASE_DIR = "agent/"
+        ORCH_URL = 'obs-load-orch.app.elastic.co:8000'
+        // Set below for local development
+        // ORCH_URL='10.0.2.2:8000'
+        DEBUG_MODE = '0' // set to '0' for production
+        // This is a placeholder server. This will change to a dummy APM Server when it is available.
+        APM_SERVER_URL = 'https://2a2bd0e2806a47e5996eeeec6d22e6df.apm.eu-west-3.aws.elastic-cloud.com:443'
```

Comment:
we usually put the APM URL and the APM token in a Vault secret

Reply:
The token is in a secret but I left the URL public. I'll change it.

Reply:
does not seem to be resolved yet? not sure if that's a nice-to-have or a requirement @kuisathaverat?

Reply:
This is my mistake. I accidentally resolved this thread. I'll update this as well.